oddbird/oddsite
# VueConf 19 blog post

## Description
- A new blog post about VueConf,  VueFinder, and Dynamic CSS.
- Updated slides/video in Dynamic CSS & Grid Systems talks

## Steps to test/reproduce
Run it locally, or check out the [Netlify-generated PR staging](https://deploy-preview-313--oddsite.netlify.com/)

## Impacted areas of website
- The latest blog post
- `/authors/miriam/` will show the video and links to talk pages
- Dynamic CSS & Grid Systems talks

* * *

## Reviewer tasks:

- [ ] Pulled branch and ran code locally
